.github/workflows/package.yml

```
@@ -155,11 +154,24 @@ jobs:
         ARGILLA_VERSION=${{ needs.build_server_docker_image.outputs.version }}
     secrets: inherit
 
+  deploy_environment:
+    name: Deploy branch environment
+    uses: ./.github/workflows/deploy-environment.yml
+    needs: build_quickstart_docker_image
+    if: |
+      !cancelled() &&
+      needs.build_quickstart_docker_image.result == 'success' &&
+      github.event_name == 'pull_request'
```

Maybe here we could add `github.event.pull_request.draft == false` something like

```suggestion
      github.event_name == 'pull_request' && github.event.pull_request.draft == false
```

---

will add once I finish my tests :)